CHANGELOG.md

```diff
@@ -6,9 +6,14 @@ All notable changes to this project will be documented in this file. The format
 
 > **Note**: Changes and additions listed here are contained in the `:edge` image tag. These changes may not be as stable as released changes.
 
+### Updates
+
+- **Tests:**
+  - Refactored helper methods for sending e-mails with specific `Message-ID` headers and the helpers for retrieving + filtering logs, which together help isolate logs relevant to specific mail when multiple mails have been processed within a single test.
+
 ### Fixes
 
-**Dovecot:**
+- **Dovecot:**
```

Comment:
Already handled in the `v13.3.1` release PR.

```suggestion
**Dovecot:**
```

It'd probably be a bit weird to squeeze this PR into a bug fix release 😅 (_it's marked for v14 too_)